nearcore/src/config.rs

```diff
@@ -964,19 +963,8 @@ pub fn init_configs(
                 CryptoHash::default(),
             );
             add_protocol_account(&mut records);
-            let shards = if num_shards > 1 {
-                ShardLayout::v1(
-                    (1..num_shards)
-                        .map(|f| {
-                            AccountId::from_str(format!("shard{f}.test.near").as_str()).unwrap()
-                        })
-                        .collect(),
-                    None,
-                    1,
-                )
-            } else {
-                ShardLayout::v0_single_shard()
-            };
+            // FIXME eagr version?
+            let shards = ShardLayout::of_num_shards(num_shards, 3);
```

Comment:
This would cause some genesis changes check to fail. Not sure if you want to update gensis_config.json or keep it as it was.

Reply:
It's fine to update the genesis_config.json (we're talking about the one in jsonrpc right?). The production networks genesis files are handled separately, this one should only be used for tests.